### ISSUE TITLE
Our next meetup displays the wrong time

### ISSUE THREAD

**Issue by zacck-zz** [color-code/colorcode-org-za#1]
# Bug 
On the homepage the next meet up section still shows a date from last year, I think this could just be an updating issue?

**Comment by njacobs5074** [comment]
I updated it in the latest push.  Can you check again?

**Comment by zacck-zz** [comment]
Yeah I can see in the code it should be showing the correct date however on the website even when I clear my cache I get the wrong date 

![screen shot 2018-03-11 at 10 54 52](https://user-images.githubusercontent.com/897731/37251544-b2b35386-251a-11e8-928a-80c72ca2ec6a.png)
I can try on a different machine completely and report

**Comment by njacobs5074** [comment]
Maybe also try it in an anonymous window and see if that fixes it?  But yeah the real fix is to get this page generating content dynamically.  I think that when we connect it to a service like Quicket, this issue will largely go away.

**Comment by zacck-zz** [comment]
checked on private the issue still persists, I'm positive the dynamic content will fix it.
